README.md

```diff
@@ -22,7 +22,8 @@ __Windows__
 * [Visual Studio 2015](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx)
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (add to PATH)
-* [Python 2.7+](https://www.python.org/downloads/) (add to PATH)
+* [Python 2.7+ Or 3.5+](https://www.python.org/downloads/) (add to PATH)
```

Comment:
@Zangetsu38 I think anyone who's going to compile RPCS3 is not going to be an absolute moron, that doesn't understand what "at least certain version or a newer one" means. If we state anything above 2.7 will work (2.7+), then that will include the latest release, if not stated otherwise. I don't see a reason to fix something, that's not wrong/broke/or in need of clarification.